### PR TITLE
feat: Update Content: Re-order fundamental, update authors, and enhance descriptions

### PR DESCRIPTION
**Pull request by elPiablo**
- [x]  re-order fundamentals into a less random list for learners 
- [x]  check/update authors and their details and  lesson titles and descriptions for all .mdx file except Fundamentals Track in: 
- [x] the following component:`LessonLayout`, 
- [x] and and in objects inside: `lessonArray` and in `return` statement inside `/tracks/index.tsx`
- [x] complete above tasks for Fundamentals track (I was breaking lots of things, so reverted to doing it Fundamentals separately)

### CODE REVIEW

**Comment by elPiablo** [comment]
I tried to get into production mode, but ended up breaking a lot of stuff - I guess by trying to update too much information in too many files in one go. So I reverted all the work I did, and started from scratch on the second commit. Sorry about the wasted time.

**Comment by elPiablo** [comment]
what a mission to do so little....

**Comment by elPiablo** [comment]
Hey folks, 
I originally had 13 tasks on this PR's list. That was too much for reviewing, and was holding me/everyone up time-wise.
I removed 7 tasks and deleted an unnecessary one, so we can merge this - this is still a meaningful and rounded PR, by the way.
I'll create new PRs with the remaining tasks.
AFAIC, this is ready to review and merge.

File: apps/academy/src/pages/tracks/fundamentals/decentralized-storage.mdx [hunk 1]
```diff
@@ -9,9 +9,9 @@ import LessonLayout from "../../../components/LessonLayout";
 
 <LessonLayout
    lessonTitle="Decentralized Storage" 
-   author="_7i7o, piablo"
-   authorPosition="Developer_DAO"
-   authorTwitter="@7i7o.eth, @Skruffster"
+   author="john-mac.eth"
+   authorPosition="@GeorgeMac510"
+   authorTwitter="@GeorgeMac510, @@pbillingsby_"
```
Comment (recepcankaya) [inline apps/academy/src/pages/tracks/fundamentals/decentralized-storage.mdx:14]: in this line, there is one author but two twitter accounts

File: apps/academy/src/pages/tracks/index.tsx [hunk 2]
```diff
@@ -5,26 +5,25 @@ import { TrackCard } from "ui";
 const TracksPage = () => {
   const tracksArray = [
     {
-      title: "Fundamentals.",
+      title: "Web3 Fundamentals",
       author: "authorName",
       imgPath: "/image16.png",
       description:
-        "This is the Fundamentals track's description space, it should explain in three or four lines which topics it contains and maybe something else.",
+        "There is a lot more to Web3 than blockchains and smart contracts. This track provides resources and insights on various fundamental tooling and infrastructure needed for the magic you create in your projects to be become a reality.",
```
Comment (recepcankaya) [inline apps/academy/src/pages/tracks/index.tsx:12]: there is a little typo in the last sentence

**Review by Markkos89 — APPROVED** [review]
Amazing job :rocket:

**Comment by elPiablo** [comment]
Cheers @recepcankaya and @Markkos89 !

**Pull request merged**